Casks/g/grads.rb

```diff
@@ -8,10 +8,7 @@
   desc "Access, manipulate, and visualise earth science data"
   homepage "http://cola.gmu.edu/grads/grads.php"
 
-  livecheck do
-    url "http://cola.gmu.edu/grads/downloads.php"
-    regex(/href=.*?grads[._-]?v?(\d+(?:\.\d+)+)-bin-darwin.*?\.t/i)
-  end
+  disable! date: "2025-11-30", because: :discontinued
```

Comment:
```suggestion
  disable! date: "2025-11-30", because: :no_longer_available
```

Sorry @chenrui333, just another quick change here to update the reason to the more appropriate one